### ISSUE TITLE
Re-enable and fix Xsl tests loading xsl from dll

### ISSUE THREAD

**Issue by sepidehMS** [dotnet/corefx#9988]
Re-write these tests so that they don't load xsl from the dll

These tests are marked with ActiveIssue

This needs to be done as a part of dev/xml branch work

**Comment by danmosemsft** [comment]
Requires more of code DOM. Once that is done, straightforward.

**Comment by sepidehMS** [comment]
Since CodeDom is not part of netstandard2.0 and so XSLTC.exe is not available on core, these tests are no longer relevant and will be removed.

**Comment by onnov72** [comment]
So the XslCompiledTransform in .NET Core  does not support the Load(type) method? I think this should be more clear in the documentation for this method as well as the documentation about the xsltc.exe
And I would assume that the method would throw a PlatformNotSupportedException instead of the FileNotFoundException: Could not load file or assembly 'System.Data.SqlXml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' I get now.
